ampersand-state.js

```diff
@@ -437,16 +454,17 @@ assign(Base.prototype, Events, {
         }, this);
     },
 
-    _getDerivedProperty: function (name, flushCache) {
```

Comment:
I know at one point or another i wanted to be able to do this forcibly. But can't remember why. If it's not being used and everything passes, then we're probably good.